lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
I'm completely sure it will conflict. `Promise.prototype._then` binds to `SequelizePromise.prototype.sequelize.options.namespace;`. `SequelizePromise.prototype.sequelize` refers to the latest instantiated instance of sequelize